docs/backbone.radio.md

```diff
@@ -0,0 +1,131 @@
+# Backbone Radio
+
+The Backbone Radio provides easy support for a number of messaging patterns for
+Backbone and Marionette. This is provided through two basic constructs:
+
+* Events - trigger events on a global object
+* Requests - a global request/reply implementation
+
+
+Radio takes these two constructs and adds the channel implementation - providing
+namespaces for events and requests. In short, Radio is a global, namespaced,
+message bus system designed to allow two otherwise unrelated objects to
+communicate and share information.
+
+Let's look at a simplified example in Marionette:
+
+```javascript
+var Mn = require('backbone.marionette');
+
+var NotificationView = Mn.View.extend({
+  channelName: 'notify',
```

Comment:
In the case that we did add the radio mixin to a view, or for an Object, the fun part of the Mn integration is the `radioEvents` and `radioRequests` such that instead of the `initialize` below you'd just have:
```js
radioEvents: {
  'show:success': 'showSuccessMessage',
  'show:error': 'showErrorMessage'
}
```

Although in this case I'd argue that, in this case, those should be `radioRequests` and not events as you are requesting something to happen.  Where-as events should strictly be things that occurred, ideally written in `verb:noun`